projects/Mallard/src/helpers/diagnostics.ts

```diff
@@ -94,7 +96,7 @@ The information below will help us to better understand your query:
 Product: Daily App
 App Version: ${version} ${buildNumber}
 Release Channel: ${isInBeta() ? 'BETA' : 'RELEASE'}
-App Edition: UK
+Last Edition Viewed: ${edition}
```

Comment:
Since `getSetting('edition')` return the current edition, change it to `Current Edition....`?